python/__init__.py

```diff
@@ -16,6 +16,33 @@ def get_version():
         return version_file.read().strip()
 
 
+def modcc():
+    import os
+    import sys
+    import subprocess
+
+    sys.exit(
+        subprocess.call(
+            [os.path.join(os.path.dirname(__file__), "bin", "modcc"), *sys.argv[1:]]
+        )
+    )
+
+
+def build_catalogue():
```

Comment:
Also ... since we call `sys.exit` here, does this mean we terminate our simulation by calling the harmless function `build_catalogue`?

Reply:
Yes. More reason to indicate these functions are not meant to be called by users.